extensions/amp-form/0.1/amp-form.js

```diff
@@ -104,6 +113,9 @@ export class AmpForm {
     /** @const @private {!../../../src/service/url-replacements-impl.UrlReplacements} */
     this.urlReplacement_ = urlReplacementsForDoc(element);
 
+    /** @private {?Promise} */
+    this.dependenciesPromise_ = null;
```

Comment:
Sticking to old name, the method name is more important as it describes the wait better.